easybuild/easyconfigs/b/Boost/Boost-1.59.0-GCC-5.2.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Boost'
+version = '1.59.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'GCC', 'version': '5.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+
+dependencies = [('bzip2', '1.0.6')]
+
+boost_mpi = False
+
+configopts = '--without-libraries=python --with-icu'
+
+toolset = 'gcc'
+
+osdependencies = [('zlib-devel', 'zlib1g-dev'), ('libicu-devel', 'icu-devel', 'libicu-dev')]
+
+parallel = 8
```

Comment:
EB will by default utilize the maximal number of cores in the current cpuset/cgroup. It you want to limit this to maximally 8, use `maxparallel`. Else this line can go.